hack/README.md

```diff
@@ -19,11 +19,11 @@ The standard workflow is performed inside a helper container to normalize the bu
 
 `$ make all` executes the full workflow.  For granular control of the workflow, several Make targets are defined:
 
-**Make Targets**
+#### Make Targets
 
 - `all`: cleans up previous build artifacts, compiles all CDI packages and builds containers
 - `clean`: cleans up previous build artifacts
-- `build`: compile all CDI binary artifacts
+- `build`: compile all CDI binary artifacts and generate controller manifest
```

Comment:
so we don't have a `build cloner` target. It seems asymmetric so is it worth a short explanation?

Reply:
added